src/app/mod.rs

```diff
@@ -88,15 +80,14 @@ pub struct App<'app> {
 }
 
 impl<'app> App<'app> {
-    pub fn new(state: state::AppState<'app>, cli: DftCli) -> Self {
+    pub fn new(state: state::AppState<'app>) -> Self {
```

Comment:
The cli arguments don't appear to be used after construction so I removed them from the structures